crates/rpc-types-anvil/src/lib.rs

```diff
@@ -0,0 +1,164 @@
+#![doc = include_str!("../README.md")]
+#![doc(
+    html_logo_url = "https://raw.githubusercontent.com/alloy-rs/core/main/assets/alloy.jpg",
+    html_favicon_url = "https://raw.githubusercontent.com/alloy-rs/core/main/assets/favicon.ico"
+)]
+#![warn(
+    missing_copy_implementations,
+    missing_debug_implementations,
+    unreachable_pub,
+    clippy::missing_const_for_fn,
+    rustdoc::all
+)]
+#![allow(missing_docs)]
```

Comment:
add follow up issue to remove this, we want docs, but don't want to block this PR on it since it's a niche-ish crate and it is blocking other work streams